### ISSUE TITLE
[EX/FE/QA] 10-10EZR: Update Breadcrumbs

### ISSUE THREAD

**Issue by alexseelig** [department-of-veterans-affairs/va.gov-team#84964]

**Comment by allanto-ah** [comment]
@alexseelig / @mkerns1 

The current breadcrumbs in staging is `Home > VA.gov home > My HealtheVet > Update your VA health benefits information`. Is this okay? The 2nd task in the description mentions only `VA.gov home > My HealtheVet > Update your VA health benefits information`

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeUMrQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--170e8e98030d192932491d03dc0d0cbbf95e0ac9/Screenshot%202024-07-08%20at%206.33.19%E2%80%AFAM.png" alt="Screenshot 2024-07-08 at 6.33.19 AM.png" />

**Comment by alexseelig** [comment]
I will defer to Matt and @IMHieuVo here

**Comment by IMHieuVo** [comment]
No, that is incorrect. The new breadcrumb does not have an independent "home",: `VA.gov home > My HealtheVet > Update your VA health benefits information`.

**Comment by mkerns1** [comment]
This is really weird because that Home link didn't appear when I implemented the changes. I have screenshots posted in this PR from when I merged and closed the PR: https://github.com/department-of-veterans-affairs/content-build/pull/2148

**Comment by mkerns1** [comment]
Something must have gotten changed after the fact. Maybe related to the push by MHV. I will investigate.

**Comment by hdjustice** [comment]
This has been corrected, closing this ticket
